README.md

```diff
@@ -45,6 +45,28 @@ If you use GLASS simulations or the GLASS library in your research, please
 [cite the original GLASS paper](https://glass.readthedocs.io/stable/user/publications.html)
 in your publications.
 
+<!-- markdownlint-disable MD046 MD013 -->
```

Comment:
Why are you disabling [MD046](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md046---code-block-style)?

Reply:
Removing that gives me this -
```
README.md:50 MD046/code-block-style Code block style [Expected: indented; Actual: fenced]
```

Reply:
I've fixed this in accc097465f910c4b303fd92335c58f21a794da0